Source/Laser/LaserProfilesImpl/LaserProfileFromTXYEFile.cpp

```diff
@@ -123,17 +122,15 @@ WarpXLaserProfiles::FromTXYEFileLaserProfile::parse_txye_file(std::string txye_f
         //Uniform grid flag
         char flag;
         inp.read(&flag, 1);
-        if(!inp) Abort("Failed to read sizes from txye file");
+        if(!inp) Abort("Failed to read grid type from txye file");
         m_params.is_grid_uniform=flag;
 
         //Grid points along t, x and y
-        auto const three_uint32_size = sizeof(uint32_t)*3;
-        char buf[three_uint32_size];
```

Comment:
One other source here were: `type name[len]` is a VLA (variable-length array) which is not available in C++. Probably the compilers interpreted the `sizeof` silently as `constexpr`.